mne/gui/_coreg_gui.py

```diff
@@ -796,10 +804,21 @@ class CoregPanel(HasPrivateTraits):
                        Item('fid_eval_str', style='readonly'),
                        Item('points_eval_str', style='readonly'),
                        '_',
-                       HGroup(Item('prepare_bem_model'),
-                              Label("Run mne_prepare_bem_model"),
-                              show_labels=False,
-                              enabled_when='can_prepare_bem_model'),
+                       VGroup(
+                           Item('scale_labels',
+                                label="Scale *.label files",
+                                enabled_when='n_scale_params > 0'),
+                           Item('copy_annot',
+                                label="Copy annotation files",
+                                enabled_when='n_scale_params > 0'),
+                           Item('prepare_bem_model',
+                                label="Run mne_prepare_bem_model",
+                                enabled_when='can_prepare_bem_model'),
+                           show_left=False,
+                           label='Scaling options',
+                           show_border=True,
+                       ),
```

Comment:
Looks like this is not Flake8?

https://travis-ci.org/mne-tools/mne-python/jobs/203536408#L3097